.github/workflows/sam_package_code.yml

```diff
@@ -107,27 +107,47 @@ jobs:
             .aws-sam/build
             README.md
 
-      - name: Set ECR Repository
-        run: echo "ECR_REPOSITORY=fhir-facade-repo" >> "$GITHUB_ENV"
+      - name: Set FHIR FACADE ECR Repository
+        run: echo "FHIR_FACADE_ECR_REPOSITORY=fhir-facade-repo" >> "$GITHUB_ENV"
 
-      - name: Set environment variables
+      - name: Set FHIR FACADE Image Tag
+        run: echo "FHIR_FACADE_IMAGE_TAG=${{ inputs.VERSION_NUMBER }}" >> "$GITHUB_ENV"
```

Comment:
I'd rather we just use these values inline rather than setting environment variables just to template them into other strings.